.github/workflows/test.yml

```diff
@@ -87,6 +87,9 @@ jobs:
       - name: Install
         run: pnpm i
 
+      - name: Dev
+        run: pnpm run dev
```

Comment:
I don't think this one should be included in the test workflow as it hangs

Reply:
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/22515951/191718113-34ae01b8-f028-46ab-aae3-ae39e130c1eb.png">
<img width="1513" alt="image" src="https://user-images.githubusercontent.com/22515951/191718541-7fb95d1b-0c76-45e7-b7bc-3a3cfb98c6ae.png">
Related to this. We got error while run the `test` command. Maybe there's a better way to handle it ?

Reply:
> <img alt="image" width="1279" src="https://user-images.githubusercontent.com/22515951/191718113-34ae01b8-f028-46ab-aae3-ae39e130c1eb.png">
> 
> <img alt="image" width="1513" src="https://user-images.githubusercontent.com/22515951/191718541-7fb95d1b-0c76-45e7-b7bc-3a3cfb98c6ae.png">
> 
> Related to this. We got error while run the ``test`` command. Maybe there's a better way to handle it ?